lib/hotwire/spark/engine.rb

```diff
@@ -21,5 +21,9 @@ class Engine < ::Rails::Engine
     initializer "hotwire_spark.install" do |application|
       Hotwire::Spark.install_into application if Hotwire::Spark.enabled?
     end
+
+    initializer "hotwire_spark.assets" do |application|
```

Comment:
I'd prefer to keep the install invocation as the last step in the engine. Could you please move this initializer above it?